cs_scanimage.py

```diff
@@ -344,23 +345,28 @@ def parse_args():
                         default=False,
                         dest="plugin",
                         required=False, help="Prints the report as json to stdout")
+    parser.add_argument('--user-agent',
+                        default="container-image-scan",
+                        type=str, dest="useragent",
+                        help="HTTP User agent to use for API calls. Default is 'container-image-scan'")
```

Comment:
If you are not going to suppress this option, then you will need to update the README to reflect the actual help usage screen that it currently shows to prevent confusion.

Reply:
ie: https://github.com/CrowdStrike/container-image-scan#usage

Reply:
done